cockatrice/src/tab_deck_editor.cpp

```diff
@@ -764,6 +771,24 @@ void TabDeckEditor::actPrintDeck()
     dlg->exec();
 }
 
+void TabDeckEditor::actExportDeckDecklist()
+{
+    DeckLoader *const deck = deckModel->getDeckList();
+    QString decklistUrlString;
+    if(deck){
+        decklistUrlString = deck->exportDeckToDecklist();
+        if(QString::compare(decklistUrlString, "", Qt::CaseInsensitive) == 0){
+            QMessageBox::critical(this, tr("Error"), tr("There are no cards in your deck to be exported"));
+            return;
+        }
+        QUrl decklistUrl = QUrl(decklistUrlString);
+        decklistUrlString = decklistUrl.toEncoded();
+        QDesktopServices::openUrl(decklistUrlString);
+    }
+    else
+        QMessageBox::critical(this, tr("Error"), tr("No deck was selected to be saved."));
```

Comment:
At least when I review PRs, I recommend the braces _every time_ for one liners... It makes it easier to understand

Reply:
We should change our style guide, since we recommend not using them ;)

Reply:
Yeah i'll work on the updates soon